monitoring-mvp/docker-compose.yml

```
@@ -0,0 +1,32 @@
+version: '2.2'
+
+services:
+
+  laconic:
+    image: gatewayfm/laconic:main
+    volumes:
+      - ./genesis.json:/root/.chibaclonkd/config/genesis.json
+      - ./config.toml:/root/.chibaclonkd/config/config.toml
+    command:
+        chibaclonkd start --gql-playground --gql-server
+    ports:
```

If the port is not needed to be accessible from outside, you can use
```suggestion
    expose:
       - 
```